testdata/scad/2D/issues/issue3262.scad

```diff
@@ -0,0 +1,7 @@
+s=2000;
+text("Text", size=s);
+translate([0,-1.5*s])
+text("Text", size=1.5*s);
+translate([0,-2*s])
```

Comment:
Should that be
```suggestion
translate([0,-2*s-1.5*s])
```
so there's no overlap of the 3 texts?

Reply:
Yes, i was just somewhat-lazily copying the test case from the original issue without changes, but that makes it looks a bit nicer.  I have pushed the update.